### PR TITLE
DEI translations- Governance and leadership

### PR DESCRIPTION
**Pull request by eyehwan**
https://github.com/chaoss/translations/issues/38

### CODE REVIEW

File: chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# 指导
+# 导师制
 
-问题：我们的指导计划在支持项目多元化和包容性方面表现如何？
+问题：我们的导师计划在支持项目多元化和包容性方面表现如何？
 
 ## 描述
 
-指导计划是发展和维持社区的重要组成部分，可以邀请新人加入社区，帮助现有成员在社区内成长，从而确保整个社区的健康状态。 通过指导计划，我们可以邀请不同的贡献者，为贡献者创造包容的环境，帮助其成长并尽可能回馈社区。
+导师计划是发展和维持社区的重要组成部分，可以邀请新人加入社区，帮助现有成员在社区内成长，从而确保整个社区的健康状态。 通过指导计划，我们可以邀请不同的贡献者，为贡献者创造包容的环境，帮助其成长并尽可能回馈社区。
```
Comment (xiaoya-Esther) [inline chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md:7]: “通过导师计划”

File: chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md [hunk 2]
```diff
@@ -22,7 +22,7 @@
 - 观察学员在指导计划期间和之后做出的贡献
```
Comment (xiaoya-Esther) [inline chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md:22]: propose：在被指导期间和之后

File: chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md [hunk 3]
```diff
@@ -22,7 +22,7 @@
...
 
 - 在指导计划期间和之后，观察学员在项目中的发展轨迹
```
Comment (xiaoya-Esther) [inline chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md:24]: 在被指导期间和之后

File: chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md (hunk 1 above)
Comment (eyehwan) [reply chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md:7]: **通过**导师计划是发展和维持社区的重要组成部分？
I prefer keeping as it is. :)

File: chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md (hunk 2 above)
Comment (eyehwan) [reply chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md:22]: "被" for me, not that neccessary. :)
I prefer keeping it unchanged.

File: chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md (hunk 3 above)
Comment (eyehwan) [reply chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md:24]: "被" for me, not that neccessary. :)
I prefer keeping it unchanged.

File: chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md (hunk 1 above)
Comment (xiaoya-Esther) [reply chinese/wg-diversity-inclusion/focus-areas/leadership/mentorship.md:7]: 这里是指后半句 “通过导师计划，我们可以邀请不同的贡献者，为贡献者创造包容的环境，帮助其成长并尽可能回馈社区。”

**Comment by xiaoya-Esther** [comment]
Thanks! LGTM now.

**Pull request merged**